CodePush.js

```diff
@@ -130,11 +130,8 @@ function getCurrentPackage() {
 }
 
 function getSDK(config) {
```

Comment:
With this simplification, is this function needed anymore? Or could we inline it within `checkForUpdate`?